eclair-core/src/test/scala/fr/acinq/eclair/wire/ChannelCodecsSpec.scala

```diff
@@ -335,7 +335,7 @@ class ChannelCodecsSpec extends AnyFunSuite {
     // let's decode the old data (this will use the old codec that provides default values for new fields)
     val data_new = stateDataCodec.decode(bin_old.toBitVector).require.value
     assert(data_new.asInstanceOf[DATA_WAIT_FOR_FUNDING_CONFIRMED].fundingTx === None)
-    assert(System.currentTimeMillis.milliseconds.toSeconds - data_new.asInstanceOf[DATA_WAIT_FOR_FUNDING_CONFIRMED].waitingSince < 3600) // we just set this timestamp to current time
+    assert(System.currentTimeMillis.milliseconds.toSeconds - data_new.asInstanceOf[DATA_WAIT_FOR_FUNDING_CONFIRMED].waitingSinceBlock < 3600) // we just set this timestamp to current time
```

Comment:
Is this comparison still valid ?

Reply:
We will still do that "trick" for upgrades from old eclair nodes, but it will then be changed again in `Channel.scala` to use a block height. I thought about directly using `currentBlockHeight` in the legacy codec, but it requires changing a `val` to a `def` to pipe the `blockCount` which was messy...